packages/kernel-6.12/kernel-6.12.spec

```diff
@@ -887,6 +887,45 @@ install -p -m 0644 %{S:301} %{buildroot}%{_cross_bootconfigdir}/05-vmware.conf
 %{_cross_kmoddir}/kernel/drivers/xen/xen-gntdev.%{_ko}
 %{_cross_kmoddir}/kernel/drivers/xen/xen-privcmd.%{_ko}
 %endif
+
+# Exclude SCSI modules for aarch64 since VMware
+# support is only available for x86_64.
+%if "%{_cross_arch}" == "aarch64"
+%exclude %{_cross_kmoddir}/kernel/drivers/block/pktcdvd.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/cdrom/cdrom.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/scsi/iscsi_boot_sysfs.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/scsi/iscsi_tcp.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/scsi/libiscsi.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/scsi/libiscsi_tcp.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/scsi/scsi_mod.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/scsi/scsi_transport_iscsi.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/scsi/scsi_transport_sas.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/scsi/sd_mod.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/scsi/virtio_scsi.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/target/loopback/tcm_loop.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/usb/storage/uas.%{_ko}
+%exclude %{_cross_kmoddir}/kernel/drivers/usb/storage/usb-storage.%{_ko}
+%endif
```

Comment:
There's a lot of duplication between the `aarch64` and `x86_64` module list. All of these can go into the "generic" set that isn't guarded by an arch check.

Reply:
These are guarded with if check because we are excluding all the modules for aarch64.